week5/ks21_transform_the_string.py

```diff
@@ -0,0 +1,27 @@
+def parse_input():
+    test_cases = []
+    for _ in range(int(input())):
+        padlock = input()
+        favorite = input()
+        test_cases.append((padlock, favorite))
+    return test_cases
+
+
+def solution(test_case):
+    padlock, favorite = test_case
+    num_op = 0
+    for c in padlock:
+        c_op = 13
+        for f in favorite:
```

Comment:
You can maybe have a lookup table to avoid computing the same character more than once.